products/spanner/api.yaml

```diff
@@ -143,3 +144,8 @@ objects:
           execute atomically with the creation of the database: if there is an
           error in any statement, the database is not created.
         input: true
+      - !ruby/object:Api::Type::String
```

Comment:
This is an enum, right?

Reply:
It's output-only, so marking it as an enum would introduce meaningless validation logic.  Accepted values currently seem to be `CREATING` and `READY` ... we're never going to return before it's `READY` ... so a datasource that just returns the string `READY` would be exactly as good.  What I'm saying is, it's not a useful value, it's only included because it's already in the resource and I didn't want to make a breaking change.